app/templates/form/responses/destroy.hbs

```diff
@@ -15,10 +15,8 @@
   </table>
 </div>
 
-<button {{action 'destroy'}} class='btn btn-danger' type='button'>Ja</button>
-<LinkTo @route='activities.index' class='btn btn-default'>
-  Nee
-</LinkTo>
+<button {{action 'destroyModel'}} class='btn btn-danger' type='button'>Ja</button>
```

Comment:
On all other destroy confirmations (articles, groups, etc.), the options are "Verwijderen" and "Annuleren", but here it is "Ja" and "Nee". Not a major issue but it is inconsistent